domain/src/test/java/org/oppia/domain/question/QuestionTrainingControllerTest.kt

```diff
@@ -122,12 +137,25 @@ class QuestionTrainingControllerTest {
   // TODO(#89): Move this to a common test application component.
   @Module
   class TestModule {
+    @Mock
+    lateinit var questionTrainingConstantsProvider: QuestionTrainingConstantsProvider
+
     @Provides
     @Singleton
     fun provideContext(application: Application): Context {
       return application
     }
 
+    @Provides
+    @Singleton
+    fun provideQuestionTrainingConstantsProvider(): QuestionTrainingConstantsProvider {
+      MockitoAnnotations.initMocks(this)
```

Comment:
This is a bit hacky and ought to be done only for test classes. However we can avoid using Mockito here altogether if you move the constant directly to the dagger graph.

Reply:
Got it, I have moved it to the dagger graph now